.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "vsicons.presets.angular": true
```

Comment:
What is this for? I assume the icons that in the component below? I am not 100% sure on how compatible these are when deployed - you can use the `<mat-icon>` element along with https://material.io/resources/icons/?style=baseline for icons that can be displayed in the application.